app/validators/csv_validator.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CSVValidator < ActiveModel::Validator
+  def validate(csv_form)
+    @csv_form = csv_form
```

Comment:
The `Validator` pattern in Rails has 1 big drawback: It acts nearly as a Singleton :harold:.
See: https://api.rubyonrails.org/v7.0.5/classes/ActiveModel/Validator.html#:~:text=Note%20that%20the%20validator%20is%20initialized%20only%20once%20for%20the%20whole%20application%20life%20cycle%2C%20and%20not%20on%20each%20validation%20run.

This means that a validator will be initialized once for the whole application lifecycle — so an instance variable might leak from one request to another (which is a security issue). For that reason, we must not use instance variables in a validator 👍